package.json

```diff
@@ -19,7 +19,7 @@
     "url": "https://github.com/prettier/prettier-vscode/issues"
   },
   "engines": {
-    "vscode": "^1.15.0"
+    "vscode": "^1.17.0"
   },
   "categories": [
     "Formatters"
```

Comment:
Going to add `multi root ready`?

https://github.com/prettier/prettier-vscode/issues/243#issuecomment-340616573

Reply:
eventually yes.

Just wanted to really test it first.